retriever/bitbucketretriever/retriever.go

```diff
@@ -0,0 +1,103 @@
+qpackage bitbucketretriever
```

Comment:
```suggestion
package bitbucketretriever
```

**issue (package name):** There is a typo in the package name that makes it impossible to build the app.